glumpy/library/build-spatial-filters.py

```diff
@@ -274,7 +274,7 @@ def __init__(self):
         SpatialFilter.__init__(self, radius=1.5)
 
     def weight(self, x):
-        if x <  0.75:
+        if x <  0.5:
             return 0.75 - x * x
```

Comment:
What about the 0.75 then, shoudl it be 0.25 to have positive result?

Reply:
No. That constant is fine. You can check by making sure it lines up with the next branch: `0.5 * (0.5 - 1.5)^2 = 0.5`, which is also what `0.75 - 0.5^2` evaluates to.